### PR TITLE
README update and sync-*.yml "fix"

### PR DESCRIPTION
**Pull request by kz6fittycent**
- Updated README to match `signal-desktop`
  - Updated `sync-version-with-upstream.yml` - seems to be an indent causing an issue, as the script is working locally

### CODE REVIEW

**Comment by kz6fittycent** [comment]
If there are anymore contributors to the snap that we want to list, please provide that list of names here and I'll add them with another commit. TY.

File: README.md [hunk 1]
```diff
@@ -2,61 +2,84 @@
   <img src="http://mirrors.jenkins.io/art/jenkins-logo/logo+title.svg" alt="Jenkins" height="256px">
 </h1>
 
-<p align="center"><b>This is the snap for Jenkins</b>, the leading open-source automation server. It works on Ubuntu, Fedora, Debian, and other major Linux
-distributions.</p>
+<p align="center"><b>This is the snap for Jenkins, the leading open-source automation server</b>. It is a community-maintained package to easily install Jenkins on Ubuntu, Fedora, Debian and other major Linux distributions. It is available in the Snap Store, Ubuntu Software, and a number of other applications.</p>
```
Comment (merlijn-sebrechts) [inline README.md:5]:
```suggestion
<p align="center"><b>This is the snap for Jenkins</b>. It is a community-maintained package to easily install Jenkins on Ubuntu, Fedora, Debian and other major Linux distributions. It is available in the Snap Store, Ubuntu Software, and a number of other applications.</p>
```

File: README.md [hunk 2]
```diff
@@ -2,61 +2,84 @@
...
+<a href="https://snapcraft.io/jenkins"><img src="https://snapcraft.io/jenkins/badge.svg" alt="Snap Status"></a>
+<a href="https://github.com/snapcrafters/jenkins/actions/workflows/sync-version-with-upstream.yml"><img src="https://github.com/snapcrafters/jenkins/actions/workflows/sync-version-with-upstream.yml/badge.svg"></a>
+<a href="https://github.com/snapcrafters/jenkins/actions/workflows/release-to-candidate.yaml"><img src="https://github.com/snapcrafters/jenkins/actions/workflows/release-to-candidate.yml/badge.svg"></a>
+<a href="https://github.com/snapcrafters/jenkins/actions/workflows/promote-to-stable.yml"><img src="https://github.com/snapcrafters/jenkins/actions/workflows/promote-to-stable.yml/badge.svg"></a>
 </p>
 
 ## Install
 
-    sudo snap install jenkins
-    xdg-open http://localhost:8080
```
Comment (merlijn-sebrechts) [inline README.md:14]: Leave this command, it's useful

File: README.md [hunk 3]
```diff
@@ -2,61 +2,84 @@
...
+git fetch --all -p
+```
 
-    systemctl status snap.jenkins.jenkins.service
+Now that your git metadata has been updated you are ready to create a bugfix branch, make your changes, and open a pull request.
 
-Set a custom configuration:
+1. All pull requests should go to the stable branch so create your branch as a copy of the stable branch:
 
-    cat config.xml | sudo /snap/bin/jenkins.config
```
Comment (merlijn-sebrechts) [inline README.md:24]: We shouldn't remove this info, imo. Put it under its own headline after `## Install`. Something like `## Snap Configuration`

File: README.md [hunk 4]
```diff
@@ -2,61 +2,84 @@
...
+6. Someone from the team will review the open pull request and either merge it or start a discussion with you with additional changes or clarification needed.
+7. Once the pull request has been merged into the stable branch, a GitHub action will rebuild the snap using your changes and publish it to the [Snap Store](https://snapcraft.io/jenkins) into the `candidate` channel. After sufficient testing of the snap from the candidate channel, one of the maintainers or administrators will promote the snap to the stable branch in the Snap Store.
 
-## The Snapcrafters
+## Maintainers
 
-| [![Evan Dandrea](http://gravatar.com/avatar/b9c6dc703231efc4e307d3c59d1f9321?s=128)](https://github.com/evandandrea/) |
-| :---: |
-| [Evan Dandrea](https://github.com/evandandrea/) |
+- [@merlijn-sebrechts](https://github.com/merlijn-sebrechts/)
```
Comment (merlijn-sebrechts) [inline README.md]: I don't think you need to add me. This is all yours ;)

**Review by merlijn-sebrechts — CHANGES_REQUESTED** [review]
In addition to the comments above: You should add a blank line after every heading. Example:

```md
## License

The license of both the build files in this repository and Jenkins itself is the [MIT License](https://github.com/jenkinsci/jenkins/blob/master/LICENSE.txt)
```

**Comment by kz6fittycent** [comment]
I've addressed those various issues. Thanks for catching them.

**Review by merlijn-sebrechts — APPROVED** [review]
LGTM!

**Pull request merged**